game-headed/src/main/resources/org/triplea/game/client/ui/javafx/fxml/AboutInformation.fxml

```diff
@@ -15,11 +15,6 @@
             <Insets top="10.0" />
          </VBox.margin>
       </Button>
-      <Button mnemonicParsing="false" onAction="#showRuleBook" styleClass="normal-font-size" text="%main.button.rule_book">
```

Comment:
You aren't supposed to know this, but this lang string is referenced in the `.lang` file we provide for all the translations.
So currently there's an unused translation string there that should be removed.

Reply:
Good call out.

It's a bit odd to see so many other properties that are unused:
![Screenshot from 2020-02-17 16-38-37](https://user-images.githubusercontent.com/12397753/74694202-047cde00-51a4-11ea-8c9a-0985d4e5c3b9.png)

Reply:
Well they aren't unused, but IntelliJ is obviously only able to find the static usages.
All the greyed out entries are the engine properties that derive their translation key based on the enum name

Reply:
Is that true for these properties as well?

```
settings.tab.ai=AI
settings.tab.battle_simulator=Battle Simulator
settings.tab.combat=Combat
settings.tab.game=Game
settings.tab.map_scrolling=Map Scrolling
settings.tab.network_proxy=Network Proxy
settings.tab.network_timeouts=Network Timeouts
settings.tab.folder_locations=Folders
settings.tab.testing=Testing
```

Using the enum name to index into a language property seems a bit counter-intuitive. I wonder if it would make sense to duplicate the enum name as a string for the sake of getting an an exlicit lang string. The decoupling of enum name  and language property is maybe not a bad thing for explicit linking.

Reply:
> Is that true for these properties as well?

I think so.

By decoupling you mean that we add a field to the enum that holds the key right so we use a hardcoded string instead of generating one.
I'm not sure if intelliJ is smart enough to realise this is the lang string so it wouldn't solve the unused problem,  AFAIK it only checks fxml files for javafx.
I agree that it would make things more "refactor-proof" even though we can change both things already without breaking anything, if we remember to change both.
Regardless this could become a little bit tricky to achieve nevertheless. See, almost every enum maps to more than one javafx node. I tried to keep the key naming scheme consistent, but there are still a lot of strings to take care of

Reply:
Yeah, by decoupling, essentially if we wanted to rename the enum, or the string property key, the two could be updated independent of one another. 

NBD at this point, was mostly spitballin'